search_events_app/tests/models/test_country.py

```diff
@@ -16,3 +21,33 @@ def test_country_basic_info(self):
 
     def test_country_str(self):
         self.assertEqual(self.country.__str__(), 'Argentina')
+
+    @patch.object(
+		Country,
+		'objects'
+	)
+    def test_get_context(self, mock_objects):
+        countries = [
+			Country(label='Argentina', code='AR', eventbrite_id='1234'),
+			Country(label='Spain', code='ES', eventbrite_id='4567')
+		]
+        
+        mock_objects.all = MagicMock(return_value=countries)
```

Comment:
Are you patching `countries = cls.objects.all()` ? Are you sure that this call return a list? Maybe return an object type _queryset_. So, you need to return an object with fake data but with the same type.